plugins/child/child.js

```diff
@@ -76,13 +76,16 @@ s.cameraVals=function(e){
 }
```

Comment:
That explains the duplicated code and why when I originally made changes there they have no effect, but yeah, it's there for later.